pkg/awsapi/sts.go

```diff
@@ -0,0 +1,44 @@
+package awsapi
+
+//go:generate mockgen -destination=mocks/mock_stsiface.go -package=mock_awsapi github.com/aws/aws-sdk-go/service/sts/stsiface STSAPI
+//go:generate mockgen -destination=mocks/mock_sts.go -package=mock_awsapi github.com/keikoproj/iam-manager/pkg/awsapi STSIface
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sts"
+	"github.com/aws/aws-sdk-go/service/sts/stsiface"
+	"github.com/keikoproj/iam-manager/pkg/log"
+)
+
+type STSIface interface {
+	GetAccountID(ctx context.Context) (string, error)
+}
+
+type STS struct {
+	Client stsiface.STSAPI
+}
+
+func NewSTS() *STS {
+	sess, err := session.NewSession(&aws.Config{Region: aws.String("us-west-2")})
```

Comment:
May be we should move region also into config map as everything else is now coming from cm and nothing is hard coded

Reply:
yup, true.